### DISCUSSION TITLE
How to Block .exe and internet access for games

### DISCUSSION THREAD

**Discussion by githubplanetarparrot** [Heroic-Games-Launcher/HeroicGamesLauncher#4035] [Q&A]
Hello,

There are some games in my library that I want to block online features for. Is there a way to block these games from accessing the internet using the launcher or prefix settings?

**Comment by arrowgent** [comment]
only way i figured was either /etc/hosts file
or using a sandbox like firejail
with firejail ive basically blocked access to the entire heroic launcher
so that all spawned children (wine/proton) also have no internet.

it would be possible to setup so that firejail is included with all wine processes, only blocking wine/proton from having internet.

**Reply by githubplanetarparrot** [reply]
My issue is only some games I have I wish to block, not all. At this time I simply use Heroic for online games and Bottles for my singeplayer games.
